app/models/vim_performance_analysis.rb

```diff
@@ -584,12 +584,10 @@ def self.calc_slope_from_data(recs, x_attr, y_attr)
     slope_arr
   end
 
-  def self.get_daily_perf(obj, range, ext_options, perf_cols)
-    return unless perf_cols
-
+  def self.get_daily_perf(obj, range, ext_options)
     ext_options ||= {}
     Metric::Helper.find_for_interval_name("daily", ext_options[:time_profile] || ext_options[:tz], ext_options[:class])
-                  .order("timestamp").select(perf_cols)
+                  .order("timestamp")
```

Comment:
Could we change this PR to just be a comment on this line that perf_cols is broken?

I think the perf_cols passed in are correct.
We just can't handle them right now.

I'm actively updating virtual column support for this purpose. (mostly sorting, but select should come for free)